spec/system/page_objects/components/topic_view.rb

```diff
@@ -10,6 +10,10 @@ def has_read_post?(post)
           wait: Capybara.default_max_wait_time * 2,
         )
       end
+
+      def has_tracking_status?(name)
```

Comment:
This is too vague, this applies to tons of elements on the page. We have SelectKit page object helpers, you should use those here. https://github.com/discourse/discourse/blob/8f78425e0cfba073b3b7c373515b9adac2ce8d41/spec/system/page_objects/components/select_kit.rb . And an example:

https://github.com/discourse/discourse/blob/8f78425e0cfba073b3b7c373515b9adac2ce8d41/spec/system/composer/default_to_subcategory_spec.rb#L18-L22